src/schema/v2/payment_device_union.ts

```diff
@@ -12,23 +12,12 @@ export const ManualPaymentType = new GraphQLObjectType({
   fields: () => ({
     isManualPayment: {
       type: new GraphQLNonNull(GraphQLBoolean),
-      description: "present in case the __typename wasnt enough",
-      resolve: (_object) => {
-        console.error("im in the resolver!")
-        console.warn(_object)
-        // throw new Error("???")
-        return true
-      },
+      resolve: () => true,
     },
   }),
 })
 
 export const PaymentDeviceUnionType = new GraphQLUnionType({
   name: "PaymentDeviceUnion",
   types: [BankAccountType, CreditCardType, ManualPaymentType],
-  resolveType: (value) => {
-    if (value.foo === 42) {
-      return ManualPaymentType
-    }
-  },
```

Comment:
I was never able to get this function firing, even when explicitly returning something without a `__typename` in the stiched resolver (see https://github.com/artsy/metaphysics/pull/4086/commits/bd59fa135dfbc2118451b7a269f627945b7e4740#diff-c1775973b51cf561f4b812a9a296de0f15a8829b65a608ad42642cb67aabf1c9L201). Likewise for the `isManualPayment` field above. So i ultimately decided to remove this. I suppose stitched resolvers are expected to provide their own type name and fields